Xamarin.Forms.Sandbox/Visual.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Xamarin.Forms;
+
+namespace SkiaSharpVisual
+{
+	public sealed class SkiaSharp : IVisual
+	{
+		public static SkiaSharp Instance { get; } = new SkiaSharp();
```

Comment:
I am not sure how we would want third parties to set up their visual. We might want to have a look at creating a patter to keep everyone consistent. But, this appears to work - even though I am creating a conflict with the `SkiaSharp` namespace...